### PR TITLE
add single article page

### PR DESCRIPTION
**Pull request by 8maziar**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -83,4 +83,22 @@ color: white;
 
 .articles-list p {
   font-size: 16px;
+}
+
+.articles-list button {
+  padding: 8px;
+}
+
+/* Article page */
```
Comment (RichardNeat) [inline src/App.css]: Please remove comments and console logs before committing :)

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Nice one Maziar, the functionality all looks good however could we consider how we lay this information out on the screen for the single article? Are we rendering enough data? I currently can't view the review body. Lets consider how we label/layout each piece of information and make it look a bit nicer :)

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
Nice work! Can you add visual feedback for the user to show when the page is loading🙂 Submit another PR when this is done please😁

**Review by mezzDavies — APPROVED** [review]
Hi Maziar this is looking good functionality-wise. I think a little work on the article page would be useful for the user - maybe find a way to not need the labels (eg Author Topic etc.). Also maybe clicking on the photo of the article on the article page could take you to the single article.

**Pull request merged**